### ISSUE TITLE
How to download ghc from a mirror?

### ISSUE THREAD

**Issue by hcoona** [tweag/rules_haskell#1665]
**Is your feature request related to a problem? Please describe.**

I suffer from slow download speed from official ghc link. I know a mirror for both ghc & stackage but didn't find an option to provide them.

**Additional context**

hackage mirror: https://mirrors.tuna.tsinghua.edu.cn/help/hackage/
stakage mirror: https://mirrors.tuna.tsinghua.edu.cn/help/stackage/

**Comment by hcoona** [comment]
OK, got it. Just use the host `~/.stack/config.yaml`

**Comment by hcoona** [comment]
No, it didn't work.

**Comment by hcoona** [comment]
Seems to be hard-coded in https://github.com/tweag/rules_haskell/blob/v0.13/haskell/gen_ghc_bindist.py

**Comment by aherrmann** [comment]
Yes, this information is hard-coded [here](https://github.com/tweag/rules_haskell/blob/v0.13/haskell/ghc_bindist.bzl).
You could use Bazel's [downloader configuration feature](https://docs.bazel.build/versions/main/command-line-reference.html#flag--experimental_downloader_config) to rewrite these URLs to the mirror you would like to use.
